docs_versioned_docs/version-ros2humble/ros/tutorials/driving.mdx

```diff
@@ -16,8 +16,9 @@ There are several ways to drive a Clearpath robot using ROS 2.
 
 Make sure your controller is paired to the robot PC. Visit the [controller pairing](../installation/controller) guide for details.
 
-With the [robot services](../installation/robot#robot-services) running, turn on your controller. Press and hold either **L1** or **R1**, and use
-the left joystick to publish a linear velocity and yaw command. The **L1** button will drive the robot at slow speeds, while **R1** will drive the robot at fast speeds.
+With the [robot services](../installation/robot#robot-services) running, turn on your controller. Press and hold
```

Comment:
Did you run Prettier on the entire repository, rather than per file? We could consider removing the repository wide Prettier instructions from the README, since it takes much longer to review a PR that updates all the files formatting, but not the content.

Reply:
Some of the changes were manual because of excessively long lines of text that made it hard to edit the files. We can wait and see if prettier causes issues repeatedly.